table/table.go

```diff
@@ -117,7 +117,8 @@ func (b block) NewIterator() *blockIterator {
 // entry.  Returns a table with one reference count on it (decrementing which may delete the file!
 // -- consider t.Close() instead).  The fd has to writeable because we call Truncate on it before
 // deleting.
-func OpenTable(fd *os.File, mode options.FileLoadingMode, cksum []byte) (*Table, error) {
+func OpenTable(
+	fd *os.File, mode options.FileLoadingMode, cksum, smallest, biggest []byte) (*Table, error) {
```

Comment:
Break it like this instead -
`func OpenTable(fd *os.File, mode options.FileLoadingMode, cksum, smallest, biggest []byte) (
*Table, error) {`